app/Services/StripePaymentService.php

```
@@ -42,4 +44,44 @@ public function createPaymentIntent(array $payload)
             return handleException($e);
         }
     }
+
+    public function createCustomer(array $payload)
+    {
+        try {
+            return Customer::create([
+                'name' => $payload['name'],
+                'email' => $payload['email'],
+            ]);
+        } 
+        catch (\Exception $e) {
+            return handleException($e);
```

use throw instead of return